rtgui/labcurve.cc

```diff
@@ -63,7 +67,28 @@ LCurve::LCurve () : FoldableToolPanel(this, "labcurves", M("TP_LABCURVE_LABEL"),
 
     avoidcolorshift = Gtk::manage (new Gtk::CheckButton (M("TP_LABCURVE_AVOIDCOLORSHIFT")));
     avoidcolorshift->set_tooltip_text (M("TP_LABCURVE_AVOIDCOLORSHIFT_TOOLTIP"));
-    pack_start (*avoidcolorshift, Gtk::PACK_SHRINK, 4);
+//    pack_start (*avoidcolorshift, Gtk::PACK_SHRINK, 4);
```

Comment:
Completely remove the `avoidcolorshift` widget too

Reply:
Done , I hope :)